app/step-functions-templates/auto_controller_sfn_template.asl.json

```diff
@@ -0,0 +1,57 @@
+{
+  "StartAt": "Receive Event",
+  "States": {
+    "Receive Event": {
+      "Type": "Pass",
+      "Next": "Job Planner"
+    },
+    "Job Planner": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::lambda:invoke",
+      "Output": "{% $states.result.Payload %}",
+      "Arguments": {
+        "FunctionName": "${__auto_job_planner_lambda_function_arn__}",
+        "Payload": "{% $states.input %}"
+      },
+      "Retry": [
+        {
+          "ErrorEquals": [
+            "Lambda.ServiceException",
+            "Lambda.AWSLambdaException",
+            "Lambda.SdkClientException",
+            "Lambda.TooManyRequestsException"
+          ],
+          "IntervalSeconds": 1,
+          "MaxAttempts": 3,
+          "BackoffRate": 2,
+          "JitterStrategy": "FULL"
+        }
+      ],
+      "Next": "For each Job"
+    },
+    "For each Job": {
+      "Type": "Map",
+      "Items": "{% $states.input.plans %}",
```

Comment:
Is plans an output of JobPlanner, see above about either making it an explicit output, or assigning plans as a variable instead